feature/security/gnsi/certz/test_data/mk_cas.sh

```diff
@@ -90,16 +90,16 @@ for  d in ${DIRS[@]}; do
 
       # Create the client and server complete certificates.
       openssl x509 -req -in ca-${d}/client-${t}-${g}-req.pem \
-        -CA ca-${d}/ca-${OFFSET}-${t}-${g}-cert.pem \
-        -CAkey ca-${d}/ca-${OFFSET}-${t}-${g}-key.pem \
+        -CA ca-${d}/ca-${OFFSET}-${t}-cert.pem \
```

Comment:
i want to say there was a reason I had this a/b versions... what problem are you solving?

Reply:
The script is failing, and I thought this may be a typo. here is a failure log. I wonder if u can run it successfully?

```
+ OFFSET=0001
+ for g in a b
+ case ${t} in
+ openssl genrsa -out ca-01/client-rsa-a-key.pem 2048
Generating RSA private key, 2048 bit long modulus (2 primes)
.........+++++
....+++++
e is 65537 (0x010001)
+ openssl genrsa -out ca-01/server-rsa-a-key.pem 2048
Generating RSA private key, 2048 bit long modulus (2 primes)
........................+++++
................................+++++
e is 65537 (0x010001)
+ openssl req -new -key ca-01/client-rsa-a-key.pem -out ca-01/client-rsa-a-req.pem -config client_cert.cnf
+ openssl req -new -key ca-01/server-rsa-a-key.pem -out ca-01/server-rsa-a-req.pem -config server_cert.cnf
+ openssl x509 -req -in ca-01/client-rsa-a-req.pem -CA ca-01/ca-0001-rsa-a-cert.pem -CAkey ca-01/ca-0001-rsa-a-key.pem -out ca-01/client-rsa-a-cert.pem -CAcreateserial -days 3650 -sha256 -extfile client_cert_ext.cnf
Signature ok
subject=C = AQ, ST = NZ, L = NZ, O = OpenconfigFeatureProfiles, CN = client.service.example.com
Can't open ca-01/ca-0001-rsa-a-cert.pem for reading, No such file or directory
140704404551424:error:02001002:system library:fopen:No such file or directory:crypto/bio/bss_file.c:69:fopen('ca-01/ca-0001-rsa-a-cert.pem','r')
140704404551424:error:2006D080:BIO routines:BIO_new_file:no such file:crypto/bio/bss_file.c:76:
unable to load certificate
+ openssl x509 -req -in ca-01/server-rsa-a-req.pem -CA ca-01/ca-0001-rsa-a-cert.pem -CAkey ca-01/ca-0001-rsa-a-key.pem -out ca-01/server-rsa-a-cert.pem -CAcreateserial -days 3650 -sha256 -extfile server_cert_ext.cnf
Signature ok
subject=C = AQ, ST = NZ, L = NZ, O = OpenconfigFeatureProfiles, CN = {bb1d2545-1e93-49a6-8d0c-761d92011fde}
Can't open ca-01/ca-0001-rsa-a-cert.pem for reading, No such file or directory
140704404551424:error:02001002:system library:fopen:No such file or directory:crypto/bio/bss_file.c:69:fopen('ca-01/ca-0001-rsa-a-cert.pem','r')
140704404551424:error:2006D080:BIO routines:BIO_new_file:no such file:crypto/bio/bss_file.c:76:
unable to load certificate
+ for g in a b
+ case ${t} in
+ openssl genrsa -out ca-01/client-rsa-b-key.pem 2048
Generating RSA private key, 2048 bit long modulus (2 primes)
.................................................................+++++
......+++++
```

Reply:
either we need to change script to create ca-0001-rsa-{a|b}-cert.pem or simply use ca-0001-rsa-cert.pem when creating client/server certificates. Am I missing something?

Reply:
@morrowc  what do u suggest?

Reply:
i think a pending PR from me fixes the broken a/b problem...

https://github.com/openconfig/featureprofiles/pull/2518

i think, actually.

Reply:
oh somewhat the same fix, I had in the other PR, but I also cleaned up somethings and removed some duplication of code.